Automattic/pocket-casts-android
# Allow large widget to be resized horizontally

## Description

This allows large widget to be resized horizontally.

## Testing Instructions

1. Add large widget on a tablet. Widget previews might look poorly at the moment.
2. You should be able to resize large widget horizontally.
3. Test that the widgets looks okay with different queue states.

## Screenshots or Screencast 

| Narrow | Wide |
| - | - |
| ![a](https://github.com/Automattic/pocket-casts-android/assets/30936061/9543c7ac-1706-4b97-9853-a715333a1318) | ![Screenshot_20240429_130836](https://github.com/Automattic/pocket-casts-android/assets/30936061/1c1650c4-5b74-495d-9665-cba0ed21b194) |

## Checklist
 
#### I have tested any UI changes...
